packages/block-library/src/navigation/block-navigation-list.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
```

Comment:
Can we use the [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here?

Reply:
Unfortunately not, because we still support IE 11.

Reply:
This is a bit confusing to me. It's stated [here](https://github.com/WordPress/gutenberg/tree/master/packages/block-library#installation), that this code will run in [envs that support ES2015+](http://kangax.github.io/compat-table/es6/), which would imply that IE11 users have to use the polyfills anyway?